src/components/analytics/analytics.model.ts

```diff
@@ -31,6 +31,6 @@ const analyticsSchema = new Schema({
 })
 
 // the latter interface contains ORM-based functions such as save().
-interface IAnalyticsModel extends IAnalytics, mongoose.Document { }
+interface IAnalyticsModel extends analyticsInterface, mongoose.Document { }
```

Comment:
Suggestion: Interfaces and Schemas always start with capital :p
The InfoInterface and InfoSchema it looks good.

Reply:
You're right :"), I will create a commit that will solve this issue, I've noticed that there're many interfaces that didn't start with capital letters

Reply:
Sure, resolving this for now